.github/workflows/test.yml

```diff
@@ -31,7 +32,12 @@ jobs:
       - name: Install dependencies
         run: |
           pip install -U pip "setuptools<79.0.0" wheel
-          SETUPTOOLS_ENABLE_FEATURES=legacy-editable pip install -r ./requirements.txt
+          SETUPTOOLS_ENABLE_FEATURES=legacy-editable pip install -r ./requirements-base.txt
```

Comment:
you can keep old `requirements.txt` and add `pip install -U "mypy==${{ matrix.mypy-version }}"`

Reply:
I wanted to avoid installing 2 versions of mypy to gain some ci time